Dockerfile

```diff
@@ -11,8 +11,9 @@ FROM node:10.12.0-alpine
 COPY --from=build-env /app/bin /app/bin
 COPY --from=build-env /app/build /app/build
 COPY --from=build-env /app/node_modules /app/node_modules
+COPY --from=build-env /app/package.json /app/package.json
```

Comment:
in this case, the `version` command ready the app version from `package.json` at runtime: https://github.com/iov-one/iov-faucet/blob/master/src/actions/version.ts#L5

Reply:
👌 I am not familiar enough with node project versioning and best practices so I would just explain an idea:
in order to be in sync with repo tags, you could dump `version=${TRAVIS_TAG:-manual}` into a file within the build container or before and copy it into the artifact instead of package.json.
We use this ENV for tagging: https://github.com/iov-one/weave/blob/master/.travis.yml#L55

Reply:
When publishing, you use the `npm` comannd line tool to bump the version in package.json and set a git tag at the same time. So docker can utilize the git tag